### PR TITLE
[3294] Inform accredited bodies of new features

### LINKED ISSUE #1071: [S] Accredited body features interruption screen
## Story

As an accredited body
I need to know that notifications are available
So that I can opt-in to receive them

```
Given I am an accredited body user of Publish
When I log into my organisation
Then I see a screen alerting me to new features that are available
```

## Success criteria 

* Only accredited body users can see this screen when they log in to Publish
* The interruption screen will **always appear** when they log in to Publish until they click "continue"
    * If they click on "view this feature" they will be taken to the accredited body course view page
    * If they click on "subscribe to notifications", they will be taken to the notifications opt-in page
    * If they click on "continue", they will be taken to their organisation homepage
* Once they have clicked continue, the interruption page will **no longer appear** when they log in to Publish
* We are able to track how 'successful' this interruption page is - by notification opt-ins and by accredited body course page hits

## Size
S

## Who knows about this

@jesseyuen2 @izzywatson3 @monicaferraro1 

## Links to research and designs

[PR for front end banner assets](https://github.com/UKGovernmentBEIS/beis-mspsds-design-prototype/pull/33/files)

https://docs.google.com/presentation/d/1i7qDV2E9CEAY2jiacOjIBYXYnZMfbupXOZ8YDUcTtTg/edit#slide=id.g715969666f_1_72

Designs:
image

### PR DESCRIPTION
**Pull request by aliuk2012**
This is just a basic page layout. We will need to do some more tweaks to
it.

Closes #1071 
### Guidance to review
login and view 
https://s121d02-3294-ptt-as.azurewebsites.net/accredited-body-new-features
### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [ ] Cleaned commit history
- [x] Tested by running locally
- [ ] Product Review

### CODE REVIEW

File: app/views/pages/accredited_body_new_features.html.erb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    <p class="govuk-body">
+      We've recently added new features to further support accredited bodies.
+    </p>
+  </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-m">Courses as an accredited bodies</h2>
```
Comment (gpeng) [inline app/views/pages/accredited_body_new_features.html.erb:13]: Should this be singular?

File: app/views/pages/accredited_body_new_features.html.erb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    <p class="govuk-body">
+      In this section, you can see:
+    </p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>who lists you are their accredited body</li>
```
Comment (gpeng) [inline app/views/pages/accredited_body_new_features.html.erb:19]: who lists you are -> who lists you as

File: app/views/pages/accredited_body_new_features.html.erb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      <li>which courses you're accredited body for</li>
```
Comment (defong) [inline app/views/pages/accredited_body_new_features.html.erb:20]: You're the accredited body for

Comment (aliuk2012) [reply app/views/pages/accredited_body_new_features.html.erb:20]: 🤦

**Comment by aliuk2012** [comment]
[[S] Accredited body features interruption screen](https://trello.com/c/R3SwvsK2/3294-s-accredited-body-features-interruption-screen)

File: app/views/pages/accredited_body_new_features.html.erb [hunk 4]
```diff
@@ -0,0 +1,42 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">New features</h1>
+
+    <p class="govuk-body">
+      We've recently added new features to further support accredited bodies.
```
Comment (fofr) [inline app/views/pages/accredited_body_new_features.html.erb:6]:
```suggestion
      We’ve recently added new features to further support accredited bodies.
```

File: app/views/pages/accredited_body_new_features.html.erb [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-m">Courses as an accredited body</h2>
+
+    <p class="govuk-body">
+      In this section, you can see:
+    </p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>who lists you as their accredited body</li>
+      <li>which courses you're the accredited body for</li>
```
Comment (fofr) [inline app/views/pages/accredited_body_new_features.html.erb:20]:
```suggestion
      <li>which courses you’re the accredited body for</li>
```

File: app/views/pages/accredited_body_new_features.html.erb [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    </ul>
+
+    <p class="govuk-body">
+      <a href="#" class="govuk-link">View this feature</a>
```
Comment (fofr) [inline app/views/pages/accredited_body_new_features.html.erb:24]: Should this be linking somewhere?

Comment (aliuk2012) [reply app/views/pages/accredited_body_new_features.html.erb:24]: Not at the moment, it is a separate ticket

**Pull request merged**